packages/fxa-payments-server/server/lib/server.js

```diff
@@ -60,17 +60,22 @@ module.exports = () => {
   const STATIC_DIRECTORY =
     path.join(__dirname, '..', '..', config.get('staticResources.directory'));
 
-  logger.info('static.directory', { directory: STATIC_DIRECTORY});
-  app.use(serveStatic(STATIC_DIRECTORY, {
-    maxAge: config.get('staticResources.maxAge')
-  }));
-
-  // TODO routes
+  const proxyUrl = config.get('proxyStaticResourcesFrom');
+  if (proxyUrl) {
+    logger.info('static.proxying', { url: proxyUrl });
+    const proxy = require('express-http-proxy');
+    app.use('/', proxy(proxyUrl));
```

Comment:
As a followup, I want to change the proxy settings to allow middleware-style pass through (I think using [userResDecorator](https://www.npmjs.com/package/express-http-proxy#userresdecorator-was-intercept-supports-promise)). Then there will be a common function applied to index.html (either proxied or loaded from disk) that adds the config (probably just inserting it textually before `</body>`). After this the static (not proxying) version will serve all the routes (but not resources) from the substituted string instead of using `serveStatic`